test/calendarUtils.spec.ts

```diff
@@ -136,7 +136,18 @@ describe('getWeekView', () => {
       color: {primary: '', secondary: ''}
     }];
 
-    const result: WeekViewEventRow[] = getWeekView({events, viewDate: new Date('2016-06-27'), weekStartsOn: 0});
+    let result: WeekViewEventRow[] = getWeekView({events, viewDate: new Date('2016-06-27'), weekStartsOn: 0});
+    expect(result).to.deep.equal([{
```

Comment:
Rather than having 2 sets of assertions in one test can you split them into 2 describe blocks like so:
```
describe('precision = days', () => {
  // old tests as they were, but with precision = 'days' set in the getWeekView function
  it('should get the correct span, offset and extends values for events that start within the week', () => {
    // test body
  });
  // rest of tests
});

describe('precision = minutes', () => {
  // new tests that assert the new behaviour
  it('should get the correct span, offset and extends values for events that start within the week', () => {
    // test body
  });
  // rest of tests
});
```